ncm-metaconfig/src/main/metaconfig/libvirtd/pan/schema.pan

```diff
@@ -0,0 +1,109 @@
+declaration template metaconfig/libvirtd/schema;
+
+include 'pan/types';
+
+type structure_libvirtd_network = {
+    'listen_tls' ? boolean = true # enabled by default
+    'listen_tcp' ? boolean = false # disabled by default
+    'tls_port' ? type_port = 16514 # port (16514) or service name
+    'tcp_port' ? type_port = 16509 # port (16509) or service name
+    'listen_addr' ? type_hostname # IPv4/v6 or hostname
+    'mdns_adv' ? boolean = true # enabled by default
+    'mdns_name' ? string # default "Virtualization Host HOSTNAME"
+};
+
+type structure_libvirtd_socket = {
+    'unix_sock_group' ? string # restricted to root by default
+    'unix_sock_ro_perms' ? string # default allows any user
+    'unix_sock_rw_perms' ? string 
+    'unix_sock_dir' ? string # directory of created sockets
+};
+
+type structure_libvirtd_authn = {
+    'auth_unix_ro' ? string with match(SELF, 'none|sasl|polkit') # default anyone
```

Comment:
add `^()$`, like `^(none|sasl|polkit)$`
but if is the same type `auth_unix_rw`, make separate type